gladier/client.py

```diff
@@ -697,5 +697,5 @@ def get_run_url(self, action_id):
                           on the current tool being run.
         :returns: Globus webapp url of a particular run of this flow.
         """
-        return 'https://app.globus.org/flows/%s/runs/%s' % (self.flow_id,action_id)
+        return 'https://app.globus.org/flows/%s/runs/%s' % (self.get_flow_id(),action_id)
```

Comment:
There's no guarantee this will work. `get_flow_id()` will return a flow id, but it won't necessarily match the run id passed in. It may register a completely new flow id which will result in a bad flow URL. 

I think we should take this method out.